melosys-eessi-app/src/main/java/no/nav/melosys/eessi/integration/pdl/dto/PDLPerson.java

```diff
@@ -6,8 +6,8 @@
 
 @Data
 public class PDLPerson {
-    private PDLNavn navn;
-    private PDLFoedsel foedsel;
+    private Collection<PDLNavn> navn;
+    private Collection<PDLFoedsel> foedsel;
```

Comment:
Kan en person være satt opp med flere fødselsdatoer?

Reply:
Sånn som det er modellert med PDL så kan det finnes flere "sannheter". Så teknisk sett; ja. 

Vet ikke hvor reelt det er med fødselsdato da, men et typisk eksempel på flere "sannheter" vil jo være statsborgerskap. Her kan det være flere kilder med flere forskjellige opplysninger. Eks EESSI (vi skal produsere til PDL etterhvert), NAV, Skatt etc.

Fag ønsker dog vi skal bruke den "siste" sannheten: https://confluence.adeo.no/display/TEESSI/Personopplysninger+-+bruk+av+PDL

Selve mappingen der kommer nok i neste PR 😄

Reply:
Etter å ha sett i dokumentasjonen til PDL, så står dette:
>Det vil på et hvert tidspunkt kun være en forekomst av fødsel per person per master. Dvs at alle endringer er korrigeringer innenfor master. F.eks kan Folkeregisteret kun ha 1 gjeldende og NAV kan ha 1 gjeldende.

Reply:
Nice, takk for god forklaring 👍